ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -156,10 +172,16 @@ Verify RHODS Installation
     ...    label_selector=app.kubernetes.io/part-of=model-registry-operator    timeout=400s
   END
 
-  IF    ("${UPDATE_CHANNEL}" == "stable" or "${UPDATE_CHANNEL}" == "beta") or "${dashboard}" == "true" or "${workbenches}" == "true" or "${modelmeshserving}" == "true" or "${datasciencepipelines}" == "true"  # robocop: disable
-    Log To Console    Waiting for pod status in ${APPLICATIONS_NAMESPACE}
-    Wait For Pods Status  namespace=${APPLICATIONS_NAMESPACE}  timeout=200
-    Log  Verified Applications NS: ${APPLICATIONS_NAMESPACE}  console=yes
+  ${trainingoperator} =    Is Component Enabled    trainingoperator    ${DSC_NAME}
+  IF    "${trainingoperator}" == "true"
+    Wait For Deployment Replica To Be Ready    namespace=${APPLICATIONS_NAMESPACE}
+    ...    label_selector=app.kubernetes.io/part-of=trainingoperator   timeout=400s
+  END
+
+  IF    "${dashboard}" == "true" or "${workbenches}" == "true" or "${modelmeshserving}" == "true" or "${datasciencepipelines}" == "true" or "${kserve}" == "true"  or "${kueue}" == "true"  or "${codeflare}" == "true" or "${ray}" == "true" or "${trustyai}" == "true" or "${modelregistry}" == "true" or "${trainingoperator}" == "true"    # robocop: disable
+      Log To Console    Waiting for pod status in ${APPLICATIONS_NAMESPACE}
+      Wait For Pods Status  namespace=${APPLICATIONS_NAMESPACE}  timeout=200
+      Log  Verified Applications NS: ${APPLICATIONS_NAMESPACE}  console=yes
   END
```

Comment:
In addition, we could check that the list of components present in the DSC correspond to the expected one. In this way we would know if automation is not handling a new component